### ISSUE TITLE
Move files using SFTP protocol is stucked

### ISSUE THREAD

**Issue by Sonic3R** [rclone/rclone#5824]
#### What is the problem you are having with rclone?
I tried to move items from remote using SFTP config, to local (Linux, Debian)

#### What is your rclone version (output from `rclone version`)
v1.56.2

#### Which OS you are using and how many bits (e.g. Windows 7, 64 bit)
Debian 10 x64

#### Which cloud storage system are you using? (e.g. Google Drive)
SFTP

#### The command you were trying to run (e.g. `rclone copy /tmp remote:tmp`)
`rclone move "my-sftp:mydata/" "/tolocal/" --verbose --progress --delete-empty-src-dirs --transfers 8`

#### A log from the command with the `-vv` flag (e.g. output from `rclone -vv copy /tmp remote:tmp`)
It is stucked in transfer files and is not 100% complete. If stop process and run again move command then there's no other things to move and finish immediately.

![image](https://user-images.githubusercontent.com/6311947/142237128-46edcf2e-a48b-4bae-8526-cad69c8989b3.png)

As you see, those `transfered items` are not removed from queue when done and other remaining items won;t be moved because of full queue list.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
Can you try v1.57.0 and also [the latest beta](https://beta.rclone.org/) please?

If you can reproduce with the beta, then please `kill -QUIT` the process when it gets stuck and post the backtrace - thanks!

**Comment by Sonic3R** [comment]
Seems that operation is successful finish after 1 hour after 100% progress is shown. I don't know why takes too long to "close" connection.

**Comment by ncw** [comment]
I don't know what is happening either!

Can you run it again, and when it is in that 1 hour period then `kill -QUIT` the process when it gets stuck and post the backtrace - thanks!

**Comment by Sonic3R** [comment]
I will do. When or where access backtrack ?

**Comment by ncw** [comment]
The backtrace will be printed to standard error so will appear in the terminal. If you set `--log-file` it will appear in there.

**Comment by Sonic3R** [comment]
There's no error, just delay and waiting to finish. It takes 1 hour. I checked/looked after error but nothing

**Comment by ncw** [comment]
When it starts waiting, I'd like you to `kill -QUIT` the process, **then** look in the log.

**Comment by joshualyon** [comment]
I have what sounds like a similar issue. After copying files with rclone, things seem to get stuck -- the process reaches 100% and no further updates appear on screen except for the statistics updating (time increasing and average upload speed speed decreasing accordingly).

Here's the backtrace as requested:
https://gist.github.com/joshualyon/30532efb6262b09a1ebb50c4ce84fc88

And here's the version information:
```
rclone v1.64.0
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.10.16.3-microsoft-standard-WSL2 (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.21.1
- go/linking: static
- go/tags: none
```

**Comment by ncw** [comment]
Here is a processed version of the backtrace

I deleted the irrelevant stuff.

This looks like it is stuck listing the local directory. The local
directory listing appears to have just frozen for the last 3 minutes.
That is these processes

> 7: IO wait [3 minutes] [Created by march.(*March).processJob in goroutine 136 @ march.go:438]

So, yes, this looks very much like a WSL kernel bug to me.

```
1: semacquire [3 minutes]
    runtime    proc.go:398           gopark(#5, 0, 0xc0, 0xc6, #130)
    runtime    proc.go:404           goparkunlock(...)
    runtime    sema.go:160           semacquire1(#127, 0x20, 1, 0, 0x6b)
    sync       sema.go:62            runtime_Semacquire(#41)
    sync       waitgroup.go:116      (*WaitGroup).Wait(#50)
    march      march.go:213          (*March).Run(#50, {#12, #94})
    sync       sync.go:892           (*syncCopyMove).run(#132)
    sync       sync.go:1136          runSyncCopyMove({#11, #21}, {#15, #48}, {#14, #79}, 0, 0, 0xaf, 0)
    sync       sync.go:1147          CopyDir(...)
    copy       copy.go:95            glob..func1.1()
    cmd        cmd.go:258            Run(1, 1, 2, #416)
    copy       copy.go:93            glob..func1(#109, {#44, 2, 0xb})
    cobra      command.go:944        (*Command).execute(#18, {#43, 0xb, 0xb})
    cobra      command.go:1068       (*Command).ExecuteC(#17)
    cobra      command.go:992        (*Command).Execute(...)
    cmd        cmd.go:570            Main()
    main       rclone.go:14          main()
    runtime    proc.go:267           main()
    runtime    asm_amd64.s:1650      goexit()
7: IO wait [3 minutes] [Created by march.(*March).processJob in goroutine 136 @ march.go:438]
    runtime    proc.go:398           gopark(*, 0xb, 0, 0, *)
    runtime    netpoll.go:564        netpollblock(#7, #3, 0)
    poll       netpoll.go:343        runtime_pollWait(*, 0x72)
    poll       fd_poll_runtime.go:84 (*pollDesc).wait(*, *, 0)
    poll       fd_poll_runtime.go:89 (*pollDesc).waitRead(...)
    poll       fd_unix.go:164        (*FD).Read(*, {*, 1, 1})
    net        fd_posix.go:55        (*netFD).Read(*, {*, *, #8})
    net        net.go:179            (*conn).Read(*, {*, 0xe9, 0})
    net        :1                    (*TCPConn).Read(*, {*, *, #19})
    fshttp     dialer.go:115         (*timeoutConn).Read(*, {*, *, #4})
    io         io.go:335             ReadAtLeast({#428, *}, {*, 1, 1}, 1)
    io         io.go:354             ReadFull(...)
    ssh        transport.go:322      readVersion({#428, *})
    ssh        transport.go:306      exchangeVersions({#427, *}, {*, 5, *})
    ssh        client.go:100         (*connection).clientHandshake(*, {*, 0x17}, *)
    ssh        client.go:83          NewClientConn({#13, *}, {*, 0x17}, #97)
    sftp       ssh_internal.go:41    (*Fs).newSSHClientInternal(#48, {#12, #95}, {#10, 3}, {*, 0x17}, 0)
    sftp       sftp.go:603           (*Fs).sftpConnection(#48, {#12, #95})
    sftp       sftp.go:694           (*Fs).getSftpConnection.func1()
    fs         pacer.go:88           pacerInvoker(#42, *, 0x30)
    pacer      pacer.go:197          (*Pacer).call(#42, *, 0xa)
    pacer      pacer.go:216          (*Pacer).Call(#42, #95)
    sftp       sftp.go:693           (*Fs).getSftpConnection(#48, {#12, #95})
    sftp       sftp.go:1884          (*Fs).stat(#48, {#12, #95}, {*, *})
    sftp       sftp.go:1895          (*Object).stat(*, {#12, #95})
    sftp       sftp.go:1183          (*Fs).NewObject(#48, {#12, #95}, {*, *})
    march      march.go:442          (*March).processJob.func3(0, {#432, *})
    march      march.go:450          (*March).processJob.func4()
    runtime    asm_amd64.s:1650      goexit()
1: semacquire [3 minutes] [Created by march.(*March).Run in goroutine 1 @ march.go:156]
    runtime    proc.go:398           gopark(#6, 0x55, 0xa0, 0x82, #117)
    runtime    proc.go:404           goparkunlock(...)
    runtime    sema.go:160           semacquire1(#78, 0x30, 1, 0, 0x6b)
    sync       sema.go:62            runtime_Semacquire(#112)
    sync       waitgroup.go:116      (*WaitGroup).Wait(#12)
    march      march.go:452          (*March).processJob(#50, {{#77, 0xb}, {#77, 0xb}, #25, 0, 0, 1})
    march      march.go:166          (*March).Run.func1()
    runtime    asm_amd64.s:1650      goexit()
20: select [3 minutes] [Created by sync.(*syncCopyMove).startCheckers in goroutine 1 @ sync.go:457]
    runtime    proc.go:398           gopark(*, 2, *, *, *)
    runtime    select.go:327         selectgo(*, *, *, 0, #105, 1)
    sync       pipe.go:126           (*pipe).GetMax(#92, {#12, #95}, *)
    sync       sync.go:334           (*syncCopyMove).pairChecker(#132, 0, 0, 0, 0)
    sync       sync.go:457           (*syncCopyMove).startCheckers.func1()
    runtime    asm_amd64.s:1650      goexit()
20: select [3 minutes] [Created by sync.(*syncCopyMove).startTransfers in goroutine 1 @ sync.go:473]
    runtime    proc.go:398           gopark(*, 2, *, *, *)
    runtime    select.go:327         selectgo(*, *, *, 0, #9, 1)
    sync       pipe.go:126           (*pipe).GetMax(#93, {#12, #95}, *)
    sync       sync.go:432           (*syncCopyMove).pairCopyOrMove(#132, {#12, #94}, 0, {#15, #48}, 0, 0)
    sync       sync.go:473           (*syncCopyMove).startTransfers.func1()
    runtime    asm_amd64.s:1650      goexit()
1: chan receive [3 minutes] [Created by march.(*March).Run in goroutine 1 @ march.go:206]
    runtime    proc.go:398           gopark(#46, 0, 0, 0, 0)
    runtime    chan.go:583           chanrecv(#45, 0, 1)
    runtime    chan.go:442           chanrecv1(0, 0)
    march      march.go:208          (*March).Run.func2()
    runtime    asm_amd64.s:1650      goexit()
19: select [3 minutes] [Created by march.(*March).Run in goroutine 1 @ march.go:156]
    runtime    proc.go:398           gopark(*, 2, *, *, *)
    runtime    select.go:327         selectgo(*, *, *, 0, *, 1)
    march      march.go:159          (*March).Run.func1()
    runtime    asm_amd64.s:1650      goexit()
```

**Comment by joshualyon** [comment]
> This looks like it is stuck listing the local directory. The local directory listing appears to have just frozen for the last 3 minutes.

Thanks for the insights. 

It completed all the base actions with starting up the process, creating the backend, making the various connections, and processing each of the files (checking their sizes and uploading them if needed). 

I'm not intimately familiar with the inner workings of rclone, but I vaguely remember seeing that rclone then does another pass of all the files after the upload. Are you suggesting that this is where it got stuck listing the local directory (eg. on the second pass)?

---

For my needs, we'll just update our deployment script to run directly in Powershell. We originally were running it in WSL for Windows machines as our prior solution relied on `rsync`, but since `rclone` has native Windows support, we can write the script in an OS agnostic way again.

That being said, if there's anything I can do to help with further testing or troubleshooting -- assuming you would even want to try to work around this issue with WSL -- just let me know. :)

```
PS C:\Users\josh> wsl --status
Default Distribution: Ubuntu
Default Version: 2

Windows Subsystem for Linux was last updated on 11/2/2022
The Windows Subsystem for Linux kernel can be manually updated with 'wsl --update', but automatic updates cannot occur due to your system settings.
To receive automatic kernel updates, please enable the Windows Update setting: 'Receive updates for other Microsoft products when you update Windows'.
For more information please visit https://aka.ms/wsl2kernel.

Kernel version: 5.10.16
```

**Comment by ncw** [comment]
> This looks like it is stuck listing the local directory. The local directory listing appears to have just frozen for the last 3 minutes. That is these processes
> 
> > 7: IO wait [3 minutes] [Created by march.(*March).processJob in goroutine 136 @ march.go:438]
> 
> So, yes, this looks very much like a WSL kernel bug to me.

I looked at the stuck backtrace again, and I think I was wrong with my previous interpretation - apologies!

The traces are all listing directories, but they are stuck trying to make new SSH connections to the SFTP server. So maybe the SFTP server is ignoring connections because rclone has made to many or maybe there is a networking problem or something like that.

So I don't think this is a WSL problem directly unless it is interfering with the networking.

**Comment by joshualyon** [comment]
Thanks for the update. The last logged line was:

```
2023-09-28 12:31:37 DEBUG : sftp://REDACTED@storage.bunnycdn.com:22//: Closing 53 unused connections
Transferred:       11.303 MiB / 11.303 MiB, 100%, 16.056 KiB/s, ETA 0s
Checks:               278 / 278, 100%
Transferred:          206 / 206, 100%
Elapsed time:      3m38.1sSIGQUIT: quit
```

Is it possible that it was actually hung-up trying to *close* connections?

**Comment by joshualyon** [comment]
My apologies for the late reply. I was just reviewing GitHub notifications and found this unread message. I tried the 1.65.0-beta (5824 fix) and it does seem to create substantially fewer connections. **I only saw 20 created connections and 20 closed at the end and did not experience the aforementioned issue when running this in WSL.**

I'll switch back to the WSL deployment approach using this 1.65.0-beta (5824) version to give it some more 'burn-in' time with further deployments and let you know if I see any regressions. 

Regardless of if this was the underlying issues or a side-effect, it certainly seems like a positive improvement! 

---

The reason I have some hesitation is I've been using the native 1.64.0 Windows rclone binary with the same flags (`--transfers 20 --checkers 20`) and it is consistently creating a large number of connections -- oftentimes closing upwards of 60+ connections at the end -- but it doesn't ever hang in Windows like it does in WSL.

I suspect that Bunny.net itself should be able to handle a fairly substantial number of parallel connections... that being said, I'm not intimately familiar with how WSL2 itself makes those connections and if there's any oddities there that could be contributing. I've had to use other weird workarounds for the WSL2 filesystem when running Docker containers for io intensive containers (eg. Mongodb), so it's certainly possible it's still a parallel connection (or filesystem) thing, but perhaps something unique to WSL2. 


**Windows 1.64.0 version**
```
rclone v1.64.0
- os/version: Microsoft Windows 11 Pro 22H2 (64 bit)
- os/kernel: 10.0.22621.2715 (x86_64)
- os/type: windows
- os/arch: amd64
- go/version: go1.21.1
- go/linking: static
- go/tags: cmount
```

**Windows Log Snippet**
```
...initialization, connections, file checks...

2023-11-20 11:11:48 INFO  : There was nothing to transfer
Transferred:              0 B / 0 B, -, 0 B/s, ETA -
Checks:               459 / 459, 100%
Elapsed time:         9.8s
2023/11/20 11:11:48 INFO  :
Transferred:              0 B / 0 B, -, 0 B/s, ETA -
Checks:               459 / 459, 100%
Elapsed time:         9.8s

2023/11/20 11:11:48 DEBUG : 560 go routines active
2023/11/20 11:11:48 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: Closing 62 unused connections
```

---

**WSL2 1.65.0-beta version**
```
rclone v1.65.0-beta.7426.b17151815.fix-5824-sftp-connections
- os/version: ubuntu 20.04 (64 bit)
- os/kernel: 5.15.133.1-microsoft-standard-WSL2 (x86_64)
- os/type: linux
- os/arch: amd64
- go/version: go1.21.2
- go/linking: static
- go/tags: none
```


**WSL 1.65.0-beta (5824) log snippet:**

```
2023/11/20 11:02:51 DEBUG : rclone: Version "v1.65.0-beta.7426.b17151815.fix-5824-sftp-connections" starting with parameters ["rclone" "--progress" "--inplace" "--size-only" "--no-traverse" "-vv" "copy" "--transfers" "20" "--checkers" "20" "./dist/" "bunny:/"]
2023/11/20 11:02:51 DEBUG : Creating backend with remote "./dist/"
2023/11/20 11:02:51 DEBUG : Using config file from "/home/josh/.config/rclone/rclone.conf"
2023/11/20 11:02:51 DEBUG : fs cache: renaming cache item "./dist/" to be canonical "/mnt/c/Users/josh/WebstormProjects/my-project/frontend/dist"
2023/11/20 11:02:51 DEBUG : Creating backend with remote "bunny:/"
2023/11/20 11:02:51 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40104->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023/11/20 11:02:52 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: Shell type "unix" from config
2023/11/20 11:02:52 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: Using root directory "/"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40118->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40126->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40140->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40146->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40156->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40158->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40170->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:53 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40176->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40184->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40190->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40204->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40210->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40220->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40226->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40232->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40244->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40252->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:54 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40268->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"
2023-11-20 11:02:55 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: New connection 172.29.206.227:40284->109.61.89.56:22 to "SSH-2.0-RebexSSH_5.0.8348.0"

...file checks, transfers, etc...

Transferred:       10.214 MiB / 10.214 MiB, 100%, 405.765 KiB/s, ETA 0s
Checks:               300 / 300, 100%
Transferred:          159 / 159, 100%
Elapsed time:        32.7s
2023/11/20 11:03:23 INFO  :
Transferred:       10.214 MiB / 10.214 MiB, 100%, 405.765 KiB/s, ETA 0s
Checks:               300 / 300, 100%
Transferred:          159 / 159, 100%
Elapsed time:        32.7s

2023/11/20 11:03:23 DEBUG : 185 go routines active
2023/11/20 11:03:23 DEBUG : sftp://ACCOUNT@storage.bunnycdn.com:22//: Closing 20 unused connections
```

**Comment by ncw** [comment]
> My apologies for the late reply. I was just reviewing GitHub notifications and found this unread message. I tried the 1.65.0-beta (5824 fix) and it does seem to create substantially fewer connections. **I only saw 20 created connections and 20 closed at the end and did not experience the aforementioned issue when running this in WSL.**

Thanks for testing.

I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.65 
 
> The reason I have some hesitation is I've been using the native 1.64.0 Windows rclone binary with the same flags (`--transfers 20 --checkers 20`) and it is consistently creating a large number of connections –oftentimes closing upwards of 60+ connections by the end– but it doesn't ever hang in Windows like it does in WSL.

Hmm, that is interesting. I suspect this is an oddity of WSL networking rather than rclone - maybe rclone was overloading it somehow by making too many connections too quickly.

I'm going to assume that was the case and close this issue, but if you find out otherwise please let me know.

**Comment by microyahoo** [comment]
hi @ncw, We encountered a similar issue when using rclone copy to upload data from local to the remote via SFTP. rclone would get stuck and make no progress.
```
rclone copy /mnt/csi-data-aly/shared/public/trafficlight/data drs-drs-backend-sftp-vepfs-share-wr:/training-pfs-shanghai/share-wr/public/trafficlight/data --log-file /var/log/drs/drs-hengyu-vllm-miss-bags-v9-0926-volc-1_20250926_105158.log --checkers 95 --transfers 75 --log-level INFO --ignore-existing --sftp-chunk-size 64Ki --no-traverse --progress --stats=5s --include-from /deeproute/drs/drs-hengyu-vllm-miss-bags-v9-0926-volc-1_20250926_105158.174
```

rclone config
```
root@drs-250928-decision-humanlabel-5-0-3-1-l9vlk:/# cat /root/.config/rclone/rclone.conf
[drs-drs-backend-sftp-vepfs-share]
shell_type = unix
host = drs-vepfs.srv.deeproute.cn
user = root
key_pem = <pem>
port = 2222
type = sftp
md5sum_command = md5sum
sha1sum_command = sha1sum
```

the backtrace of `kill -QUIT <pid>` as belows.
[drs-hengyu-sync-tl-label-0922-volc-1_20250922_215422.log](https://github.com/user-attachments/files/22581550/drs-hengyu-sync-tl-label-0922-volc-1_20250922_215422.log)

rclone version: `v1.70.0-beta.8791.9d464e8e9`

**Comment by ncw** [comment]
@microyahoo not sure that is the same problem. Your problem might have been fixed by 4fc4898287c73c166e09cafdf127bf6e3e382d98

Can you try to reproduce with [the latest beta](https://beta.rclone.org/) please and if you can still reproduce the problem then open a [new issue](https://github.com/rclone/rclone/issues/new/choose) on Github - with all the details - thanks

**Comment by microyahoo** [comment]
> [@microyahoo](https://github.com/microyahoo) not sure that is the same problem. Your problem might have been fixed by [4fc4898](https://github.com/rclone/rclone/commit/4fc4898287c73c166e09cafdf127bf6e3e382d98)
> 
> Can you try to reproduce with [the latest beta](https://beta.rclone.org/) please and if you can still reproduce the problem then open a [new issue](https://github.com/rclone/rclone/issues/new/choose) on Github - with all the details - thanks

@ncw I will try it and give you feedback ASAP, thanks for your quick response!

**Comment by microyahoo** [comment]
> > [@microyahoo](https://github.com/microyahoo) not sure that is the same problem. Your problem might have been fixed by [4fc4898](https://github.com/rclone/rclone/commit/4fc4898287c73c166e09cafdf127bf6e3e382d98)
> > Can you try to reproduce with [the latest beta](https://beta.rclone.org/) please and if you can still reproduce the problem then open a [new issue](https://github.com/rclone/rclone/issues/new/choose) on Github - with all the details - thanks
> 
> [@ncw](https://github.com/ncw) I will try it and give you feedback ASAP, thanks for your quick response!

hi @ncw, after three weeks of running, the stuck issue has not recurred. Thank you for your support.
